pkg/auth/auth.go

```diff
@@ -158,7 +157,7 @@ func checkFileToken(client *api.Client, tokenPath string) bool {
 func RenewService(client *api.Client, config cfg.Config) {
 	interval := time.Second * time.Duration(config.RenewalInterval)
 	log.Println("Starting the token renewer service on interval", interval)
-	ticker := time.Tick(interval)
+	ticker := time.NewTicker(interval).C
```

Comment:
Unnecessary change. No need to shut down the ticker, as we expect it to run until the program exits.

Reply:
I guess you're right. The reason to use `NewTicker()` was [the go documentation on `Tick()`, saying it leaks](https://github.com/golang/go/blob/a3985e400f1afe6f6640b03dceb4ef8396a2763a/src/time/tick.go#L52).
However even in that instance we would't need to store the `Ticker` object and run `ticker.Stop()` somewhere for it to be useful. So since it's an endless loop, let's just go back to how it was.

Reply:
Yep, precisely. For better or worse, we're going to "leak" until the program exits - but I think that as long as you're not stopping the ticker, the GC is able to handle things appropriately, so I'm not precisely sure that it actually causes issues in our use case.